src/software/ai/hl/stp/tactic/cherry_pick_tactic.cpp

```diff
@@ -12,6 +12,8 @@ CherryPickTactic::CherryPickTactic(const World& world, const Rectangle& target_r
     pass_generator.setTargetRegion(target_region);
 }
 
+
```

Comment:
Nit:

Looks like these are unnecessary

Reply:
^ bumping this comment